datasketches-memory-java8/src/main/java/org/apache/datasketches/memory/internal/AllocateDirectWritableMap.java

```diff
@@ -291,15 +274,15 @@ StepBoolean getValid() {
     }
 
     @Override
-    public void run() throws MemoryCloseException {
+    public void run() throws IllegalStateException {
       deallocate(true);
     }
 
-    boolean deallocate(final boolean calledFromCleaner) throws MemoryCloseException {
+    boolean deallocate(final boolean calledFromCleaner) throws IllegalStateException {
       if (valid.change()) {
         if (calledFromCleaner) {
           // Warn about non-deterministic resource cleanup.
-          LOG.warning("A WritableMapHandleImpl was not closed manually");
+          //LOG.warning("A WritableMapHandleImpl was not closed manually");
```

Comment:
The same question applies here - do the exceptions now convey the same information that the log message warns of?

Reply:
Same comment as the one above.